cli/src/semgrep/config_resolver.py

```diff
@@ -296,10 +292,13 @@ def _fetch_semgrep_cloud_platform_scan_config(self) -> ConfigFile:
         """
         state = get_state()
 
-        products = [
-            out.Product.from_json(PRODUCT_NAMES[p])
-            for p in self._config_path.split(",")
-        ]
+        if is_policy_id(self._config_path):
```

Comment:
Would it be easier to do something like

```
if is_policy_id(self._config_path):
  self._config_path = 'code'
```

and then leave the rest of the code untouched?

Reply:
I _just_ pushed a commit up [5baf035](https://github.com/semgrep/semgrep/pull/9225/commits/5baf0351ae6033a47c147099a26601ef32ca44c0) that treats policy the same as code. Took a different approach though:

```
PRODUCT_NAMES = {
    "code": "sast",
    "policy": "sast",
    "secrets": "secrets",
    "supply-chain": "sca",
}
```

Just added policy under products_names (even though it's not a product like that rest of the options).

Reply:
That works, my thinking was that your approach requires the rest of the code understand what policy is and makes the code a little harder to reason about than just treating policy as an alias for code on the way in and otherwise removing the implementation detail.

Reply:
Totally agree. That became apparent to me once I started fixing the tests to support `policy`. Much easier to just treat it as an alias. Added a comment in [400f6e6](https://github.com/semgrep/semgrep/pull/9225/commits/400f6e6d644c020b383b391fec595564a9da7884)

Reply:
Totally agree. That became apparent to me once I started fixing the tests to support `policy`. Much easier to just treat it as an alias. Added a comment in [400f6e6](https://github.com/semgrep/semgrep/pull/9225/commits/400f6e6d644c020b383b391fec595564a9da7884)